Website/src/rendering/src/helpers/DiscoverHelper.ts

```diff
@@ -0,0 +1,2 @@
+export const isDiscoverEnabled =
```

Comment:
This could be moved into DiscoverService.ts to be consistent with the other integrations (Boxever and Authentication for example)